build/common.ps1

```diff
@@ -197,6 +197,11 @@ Function Install-DotnetCLI {
     $msbuildExe = 'C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\bin\msbuild.exe'
     $CliTargetBranch = & $msbuildExe $NuGetClientRoot\build\config.props /v:m /nologo /t:GetCliTargetBranch
 
+    # tests depend on `dotnet list package`, which is in CLI 2.2.1xx, but hasn't yet been merged to CLI master.
+    # delete this once dotnet list package is available in CLI master
+    Trace-Log "Forcing 2.2.1xx for dotnet list package. Delete this when '$CliTargetBranch' supports dotnet list package."
```

Comment:
Does master contain the changes? 
Then it can be immediately changed in dev. 

also tracking issue for TODOs.

Reply:
I haven't checked in a while, and list package is now available in the master branch, so I'll need to remove this block when cherrypicking to the dev branch.